frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.tsx

```diff
@@ -589,7 +605,10 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
             },
             {
               key: "measureSkeleton",
-              onClick: () => this.handleMeasureSkeletonLength(tree.treeId, tree.name),
+              onClick: () => {
+                this.handleMeasureSkeletonLength(tree.treeId, tree.name);
```

Comment:
```suggestion
```

Reply:
Oops, I deleted the wrong line :see_no_evil: I simply wanted to remove the auto-close behavior for the color-changing menu.

Reply:
okay, I did that!